webforces/server/tests.py

```diff
@@ -15,7 +15,6 @@ def test_core_is_proper_singletone(self):
 class DBTest(TestCase):
     def setUp(self):
         self.core = Core(validation=True)
-        self.core.db._populateIds()
```

Comment:
We need to reset database to the initial state. That's why we need to repopulate ID database. Why you have removed `_populateIds` call and `dropAll` in tearDown is still there? Note: dropAll drops IDs database.

Reply:
Someone has added self._populateIds() to connect() method. So, _populateIds() method is called twice.

Reply:
Answered in separate review thread about `_is_done` flag.

Reply:
Done.